README.md

```diff
@@ -27,48 +27,65 @@ We use [yarn](https://yarnpkg.com) in our infrastructure, so we decided to go wi
 Please install yarn globally if you haven't already.
 
 ### Environment variables
+
 The app grabs environment variables from the `.env` file. Copy our template to your own local file:
+
 ```
 cp .env.example .env
 ```
 
 To execute transactions, you'll need to create an [Infura](https://infura.io) project and set the project ID in the `.env` you've just created:
+
 ```
 REACT_APP_INFURA_TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 ```
+
 Once done, you'll need to restart the app if it's already running.
 
 ### Installing and running
 
 Install dependencies for the project:
+
 ```
 yarn install
 ```
 
 To use the Rinkeby services:
+
 ```
 yarn start
 ```
 
 If you prefer using the Mainnet ones:
+
 ```
 yarn start-mainnet
 ```
 
+If you prefer to use Docker:
+
+```
+yarn start:docker
```

Comment:
I guess they'd have to install yarn for that first.
Maybe we should paste the plain docker command here?

Reply:
done, added `docker-compose build && docker-compose up` instead of `yarn start:docker` to the README.md